Hackathon_Converter/MainForm.Designer.cs

```diff
@@ -28,77 +84,1070 @@ protected override void Dispose(bool disposing)
+            this.label6.Location = new System.Drawing.Point(440, 364);
+            this.label6.Name = "label6";
+            this.label6.Size = new System.Drawing.Size(260, 29);
+            this.label6.TabIndex = 2;
+            this.label6.Text = "USFM File Converter";
+            this.label6.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
+            // 
+            // label5
+            // 
+            this.label5.Font = new System.Drawing.Font("Microsoft Sans Serif", 30F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.label5.Location = new System.Drawing.Point(488, 318);
+            this.label5.Name = "label5";
+            this.label5.Size = new System.Drawing.Size(154, 46);
+            this.label5.TabIndex = 1;
+            this.label5.Text = "Matos";
+            this.label5.TextAlign = System.Drawing.ContentAlignment.TopCenter;
+            // 
+            // pictureBox2
+            // 
+            this.pictureBox2.BackColor = System.Drawing.Color.Transparent;
+            this.pictureBox2.Image = ((System.Drawing.Image)(resources.GetObject("pictureBox2.Image")));
+            this.pictureBox2.Location = new System.Drawing.Point(510, 212);
+            this.pictureBox2.Name = "pictureBox2";
+            this.pictureBox2.Size = new System.Drawing.Size(98, 103);
+            this.pictureBox2.SizeMode = System.Windows.Forms.PictureBoxSizeMode.StretchImage;
+            this.pictureBox2.TabIndex = 0;
+            this.pictureBox2.TabStop = false;
             // 
             // MainForm
             // 
-            this.AutoScaleDimensions = new System.Drawing.SizeF(12F, 25F);
+            this.AutoScaleDimensions = new System.Drawing.SizeF(6F, 13F);
             this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
-            this.ClientSize = new System.Drawing.Size(800, 450);
-            this.Controls.Add(this.fileDataGrid);
+            this.ClientSize = new System.Drawing.Size(1077, 609);
             this.Controls.Add(this.panel1);
+            this.Icon = ((System.Drawing.Icon)(resources.GetObject("$this.Icon")));
+            this.Margin = new System.Windows.Forms.Padding(2);
             this.Name = "MainForm";
-            this.Text = "Hackathon Converter";
-            this.panel1.ResumeLayout(false);
+            this.Text = " Matos";
+            this.ConversionPanel.ResumeLayout(false);
+            this.ConversionPanel.PerformLayout();
+            ((System.ComponentModel.ISupportInitialize)(this.pictureBox4)).EndInit();
+            ((System.ComponentModel.ISupportInitialize)(this.pictureBox5)).EndInit();
             ((System.ComponentModel.ISupportInitialize)(this.fileDataGrid)).EndInit();
+            this.ConversionBanner.ResumeLayout(false);
+            this.HomeCapture.ResumeLayout(false);
+            ((System.ComponentModel.ISupportInitialize)(this.HomeImg)).EndInit();
+      
```

Comment:
These also should in the other source file where they are actually used. It makes things a whole lot more readable.